marathon/datadog_checks/marathon/marathon.py

```diff
@@ -57,7 +57,7 @@ def check(self, instance):
         self.process_deployments(url, timeout, auth, acs_url, ssl_verify, instance_tags)
         self.process_queues(url, timeout, auth, acs_url, ssl_verify, instance_tags)
 
-    def refresh_acs_token(self, auth, acs_url):
+    def refresh_acs_token(self, auth, acs_url, tags=[]):
```

Comment:
setting a mutable object (a list in this case) as a default value for an argument might cause problems (see http://effbot.org/zone/default-values.htm).
This should be something like:
```
def refresh_acs_token(self, auth, acs_url, tags=None):
    if tags is None:
        tags = []
```